common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
-                        "long_description": _("Download the searchable Studio reference documentation in PDF form."),
+                        "short_description": _("Clearly indicate off-platform content"),
+                        "long_description": _(
+                            "If you've linked learners to third-party tools outside of the edX course, ensure that you've"
+                            "clearly communicated that this linked content is outside of the edX platform and any terms of"
+                            "service of the third-party service apply."
+                        ),
                         "is_checked": False,
-                        "action_url": "http://files.edx.org/Getting_Started_with_Studio.pdf",
-                        "action_text": _("Download Documentation"),
-                        "action_external": True,
+                        "action_url": "CourseOutline",
+                        "action_text": _("View Course Outline"),
+                        "action_external": False,
                     },
                 ],
             },
             {
-                "short_description": _("Draft Your Course About Page"),
+                "short_description": _("Learner Engagement"),
                 "items": [
                     {
-                        "short_description": _("Draft a Course Description"),
+                        "short_description": _("Send welcome email"),
+                        "long_description": _(
+                            "You agree to send a welcome email to learners on the course start date."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Send emails throughout course run"),
+                        "long_description": _(
+                            "You agree to send paced emails (recommended: once a week) to students throughout the course run" 
+                            "to communicate when new content is released, to update on course events, etc."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Send closing email"),
                         "long_description": _(
-                            "Courses on edX have an About page that includes a course video, description, and more. "
-                            "Draft the text students will read before deciding to enroll in your course."
+                            "You agree to send a closing email at the conclusion of the course to communicate final course details."
```

Comment:
Near the end date of the course, you agree to send a closing email message to learners with final course details.